### PR TITLE
Add doctests for std::iter::Sum

### PR DESCRIPTION
**Pull request by grtlr**
Three simple doctests to test the `std::iter::Sum` trait. They cover the static, dynamic and hybrid (`DVector`) case.

### CODE REVIEW

File: src/base/ops.rs [hunk 1]
```diff
@@ -374,6 +374,46 @@ componentwise_binop_impl!(Sub, sub, ClosedSub;
                           SubAssign, sub_assign, sub_assign_statically_unchecked, sub_assign_statically_unchecked_mut;
                           sub_to, sub_to_statically_unchecked);
 
+/// Matrices and vectors implement the `iter::Sum` trait. 
+/// 
+/// # Static matrices
+///
+/// ```.rust
```
Comment (jswrenn) [inline src/base/ops.rs]: You can drop the `.rust`. Unless a different language is explicitly specified, all code comments default to being interpreted as rust code.

File: src/base/ops.rs [hunk 2]
```diff
@@ -374,6 +374,46 @@ componentwise_binop_impl!(Sub, sub, ClosedSub;
...
+///use nalgebra::Matrix3x2;
```
Comment (jswrenn) [inline src/base/ops.rs]: In nalgebra's doctests, it's been customary to hide the imports from the rendered code snippets; it helps reduce visual noise when they're rendered as html. To do this, change this import to:
```
#use nalgebra::Matrix3x2;
```

**Review by jswrenn — CHANGES_REQUESTED** [review]
Thanks! I have a few requested changes to bring the doctests in line with nalgebra's other doctests.

(The comments are on the static matrices test, but apply to the other tests, too.)

**Comment by grtlr** [comment]
That makes a lot of sense! I have formatted the doctests accordingly. I have also added a space behind `///` for better readability. The other tests of nalgebra seem to do it that way as well.

**Review by jswrenn — APPROVED** [review]

**Pull request merged**